playbooks/keepalived.yaml

```diff
@@ -47,9 +47,32 @@
       vars:
         keepalived_role: MASTER
         kp_opposite_ip: "{{ hostvars[groups['keepalived-slave'][0]].ansible_host }}"
-        notify_down_action: ""
+        notify_down_action: "/etc/keepalived/master_notify.sh"
         local_interface: "{{ local_default_interface.stdout }}"
 
+    - name: Copy necessary scripts
+      copy: src={{ item.src }} dest={{ item.dest }} mode=0755
+      with_items:
+        - { src: "{{ labkeeper_config_git_dest }}/keepalived/master_notify.sh.j2", dest: "/etc/keepalived/master_notify.sh" }
+        - { src: "{{ labkeeper_config_git_dest }}/keepalived/notify_upstream.py", dest: "/etc/keepalived/notify_upstream.py" }
+      vars:
+        upstream_user: moo-ai
```

Comment:
why moo-ai?

Reply:
As we just test , should we add another user "openlab-ci" or "allinone"?  For now, we just test, the formal account should be added at the end of publish the tool.

Reply:
OK, add a todo is better.

Reply:
@liu-sheng yeah, on master host. kp decide which deployment is failed not just based on the host poweroff, also based on the service status. The file name "notify_master" is just used in the case "MASTER ALIVE, BACKUP DOWN".